qiskit/ignis/mitigation/measurement/circuits.py

```diff
@@ -76,35 +85,44 @@ def complete_meas_cal(qubit_list=None, qr=None, cr=None, circlabel=''):
     return cal_circuits, state_labels
 
 
-def tensored_meas_cal(mit_pattern=None, qr=None, cr=None, circlabel=''):
+def tensored_meas_cal(mit_pattern: List[List[int]] = None,
+                      qr: List[QuantumRegister] = None,
+                      cr: List[ClassicalRegister] = None,
+                      circlabel: str = ''
+                      ) -> Tuple[List[QuantumCircuit], List[List[int]]
+                                 ]:
     """
     Return a list of calibration circuits
 
     Args:
-        mit_pattern (list of lists of integers): Qubits to perform the
-        measurement correction on, divided to groups according to tensors.
-        if None and qr is given then assumed to be performed over the entire
-        qr as one group
+        mit_pattern: Qubits on which to perform the
+            measurement correction, divided to groups according to tensors.
+            If `None` and `qr` is given then assumed to be performed over the entire
+            `qr` as one group (default `None`).
 
-        qr (QuantumRegister): A quantum register. If none one is created
+        qr: A quantum register. If `None`, one is created (default `None`).
 
-        cr (ClassicalRegister): A classical register. If none one is created
+        cr: A classical register. If `None`, one is created (default `None`).
 
         circlabel: A string to add to the front of circuit names for
-        unique identification
+            unique identification (default ' ').
 
     Returns:
-        A list of two QuantumCircuit objects containing the calibration
-        circuits
+        A list of two QuantumCircuit objects containing the calibration circuits
         mit_pattern
 
     Additional Information:
         The returned circuits are named circlabel+cal_XXX
         where XXX is the basis state,
-        e.g., cal_000 and cal_111
+        e.g., cal_000 and cal_111.
 
         Pass the results of these circuits to the TensoredMeasurementFitter
-        constructor
+        constructor.
+
+    Raises:
+        QiskitError: if both `mit_pattern` and `qr` are None.
+        QiskitError: if a qubit appears more than once in `mit_pattern`.
+
```

Comment:
May be nice to add an Example at the end, see: https://github.com/Qiskit/qiskit-iqx-tutorials/blob/master/qiskit/advanced/ignis/4_measurement_error_mitigation.ipynb

code-block:
```
qr = qiskit.QuantumRegister(5)
mit_pattern = [[2],[3,4]]
meas_calibs, state_labels = tensored_meas_cal(mit_pattern=mit_pattern, qr=qr, circlabel='mcal')
```

The output `state_labels` is:
`[000, 010, 101, 111]`

Reply:
same here